modules/gateway/rpc.go

```diff
@@ -10,8 +10,11 @@ import (
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+// rpcID is an 8-byte signature that is added to all RPCs to tell the gatway
+// what to do with the RPC. Empty elements of rpcID will be encoded as spaces.
```

Comment:
the encoding comment makes more sense next to the `String` method. It might actually be misleading, because empty bytes are encoded as `0x00` on the wire. The space encoding is only to make log messages more readable.